src/lighteval/models/nanotron/nanotron_model.py

```diff
@@ -343,7 +337,12 @@ def tok_decode(self, tokens: torch.LongTensor) -> List[str]:
         return self.tokenizer.batch_decode(tokens, skip_special_tokens=True)
 
     def _model_call(self, inputs: torch.Tensor) -> torch.Tensor:
-        return self.model(inputs)
+        position_ids = (
```

Comment:
Add a comment to explain why it's needed (I'm a bit curious about the int32 dtype?)

Reply:
cc @NouamaneTazi